README.md

```diff
@@ -334,6 +334,92 @@ The validation tests can be run with `ostk-validate` from within the dev contain
 | Mathematics | `main`    | Apache License 2.0 | [github.com/open-space-collective/open-space-toolkit-mathematics](https://github.com/open-space-collective/open-space-toolkit-mathematics) |
 | Physics     | `main`    | Apache License 2.0 | [github.com/open-space-collective/open-space-toolkit-physics](https://github.com/open-space-collective/open-space-toolkit-physics)         |
 
+## Using OSTk in your applications
```

Comment:
Shouldn't this be added to the base repo, since it's not so specific to astro?

Reply:
Ah yeah good point. Will add it to the base repo.